app/src/main/java/org/fossasia/phimpme/gallery/activities/LFMainActivity.java

```diff
@@ -1999,6 +2008,11 @@ protected void onPostExecute(Boolean result) {
                                     clearSelectedPhotos();
                                     getfavouriteslist();
                                     new FavouritePhotos().execute();
+                                    if(succ) {
```

Comment:
Instead of checking for the same condition -"if(succ)", every time in each of the if and else conditions, you can check it for once only. That will also do the work and the code will be less.

Reply:
The function declares true to variable succ after the function ends, so even if it isn't deleted it would make "succ=true".Thats why I did like that.

Reply:
Can you specify the line that make succ = true even the file is not deleted? Because in my view succ variable can have both values false and true. Only if we delete the file from Favourite section the succ variable is assigned the value true in background. I am just saying that for now just write once that if and else condition of yours and remove that string from strings.xml that says photo deleted from favourites as there is already an issue regarding unfavourite option.

Reply:
@Ananthusubramanian you are putting that block into three places. It can be one. Right? That's what @raj10071997 suggested. Can you do that? :)

Reply:
Yeah I will do that :)

Reply:
Okay thanks :)